substrate/bin/utils/chain-spec-builder/tests/test.rs

```diff
@@ -28,6 +31,53 @@ const DUMMY_PATH: &str = "fake-runtime-path";
 
 const OUTPUT_FILE: &str = "/tmp/chain_spec_builder.test_output_file.json";
 
+const RUNTIME_PATH: &str = unwrap_option_str(substrate_test_runtime::WASM_BINARY_PATH);
```

Comment:
Follow-up: 

there is some code in multiple places in polkadot-sdk that reads the cargo metadata, finds the target dir, and from there, tries to find the wasm path. 

If we have this new const exported by `substrate-wasm-builder`, it can likely simplify that code. 

Example from my code: 

https://github.com/iulianbarbu/polkadot-sdk/blob/8b4884dc8a9d60d09d61b08ada0e645121c0d6b8/docs/sdk/src/guides/your_first_node.rs#L146-L159

cc @pepoviola and @pgherveou I think I have seen this in your recent PRs.